### PR TITLE
Revise No Antique Firearms mod

### PR DESCRIPTION
**Pull request by Cyrano7**
Made a clearer cutoff peroid. Added some suggestions from https://github.com/CleverRaven/Cataclysm-DDA/pull/18522 and a few missed things.

### CODE REVIEW

**Comment by illi-kun** [comment]
I like and support the proposed changes. Let's hear other opinions.

**Comment by BorkBorkGoesTheCode** [comment]
The SKS shouldn't be blacklisted during this period.

**Comment by Cyrano7** [comment]
@BorkBorkGoesTheCode Its right on the border, they were being used during WW2 and started to be replaced by 1949.

**Comment by BorkBorkGoesTheCode** [comment]
>@BorkBorkGoesTheCode Its right on the border, they were being used during WW2 and started to be replaced by 1949. 

What criteria will you use to handle edge cases like these?

**Comment by Cyrano7** [comment]
Id say the sks is more on the pre-cold war side.

**Comment by Mecares** [comment]
I think the "antique" is be a bit misleading since i would think of really old firearms in that case. Wouldn't be "no pre coldwar era firearms mod" much clearer?

**Comment by Cyrano7** [comment]
Well I like the name and I think it fits well enough. But I made the description even MORE clearer to clear up any remaining confusion.

File: data/mods/No_Old_guns/modinfo.json [hunk 1]
```diff
@@ -46,12 +41,15 @@
         "44army",
         "rifle_flintlock",
         "carbine_flintlock",
+        "carbine_flintlock_double",
         "pistol_flintlock",
         "pistol_pepperbox",
+        "blunderbuss",
+        "blun_slug",
+        "blun_shot",
+        "blun_flechette"
```
Comment (mugling) [inline data/mods/No_Old_guns/modinfo.json]: Missing separator

Comment (Cyrano7) [inline data/mods/No_Old_guns/modinfo.json]: Fixed.

**Pull request merged**